### PR TITLE
New styling for forms

### PR DESCRIPTION
**Pull request by rupurt**
- New Job
- New Bridge
- Edit Bridge
- Show Bridge within same card styling as forms

[Finishes #162392533]

### CODE REVIEW

**Comment by NavyAdmiral** [comment]
To stay true to the design shouldn't the outlined button be `bare` and have `JSON Blob` as a header above?

Wouldn't it be better for consistency to also use `outlined` variant on Bridge form?

File: gui/src/components/Jobs/Form.js [hunk 1]
```diff
@@ -37,33 +37,34 @@ const Form = ({
...
+              rows={10}
+              rowsMax={25}
+              placeholder='Paste JSON'
+              multiline margin='normal'
+              name='json'
+              id='json'
+              variant='outlined'
+            />
+          </Grid>
+          <Grid item xs={12}>
```
Comment (NavyAdmiral) [inline gui/src/components/Jobs/Form.js]: Is Grid needed here? 🤔

File: gui/src/containers/Bridges/Edit.js [hunk 2]
```diff
@@ -39,53 +39,52 @@ export const Edit = useHooks(props => {
...
-                url={bridge.url}
-                confirmations={bridge.confirmations}
-                minimumContractPayment={bridge.minimumContractPayment}
-                onSuccess={SuccessNotification}
-                onError={ErrorMessage}
-              />
-            </PaddedCard>
-          ))}
+            <CardContent>
+              {onLoad(({bridge}) => (
```
Comment (NavyAdmiral) [inline gui/src/containers/Bridges/Edit.js]: Is there a possibility that `onLoad` returns false? If so, what about accomodating that scenario with a text?

File: gui/src/components/Jobs/Form.js (hunk 1 above)
Comment (rupurt) [reply gui/src/components/Jobs/Form.js]: Proably not required

**Comment by rupurt** [comment]
👍 to the outlined Bridge + label. Was meant to apply it to both.

**Pull request merged**